### ISSUE TITLE
Allow specifying if subfolders of each blacklisted folders should be sorted

### ISSUE THREAD

**Issue by Gitoffthelawn** [ariacorrente/simple-sort-bookmarks#5]
Thanks for the extension!

The blacklisting of folders works well, but what's really needed is to specify if the subfolders of each folder should be blacklisted or sorted.

At the least, a global checkbox setting: (Blacklist subfolders of blacklisted folders*

**Comment by ariacorrente** [comment]
A little more difficult than #6 , i need to see if is simple enough to be added in this version or if to delay the feature to the web extensions rewrite.

**Comment by Gitoffthelawn** [comment]
I agree it's a bit more challenging than #6.  Note that Auto-Sort Bookmarks extension has a very elegant way to handle this (once you drill down to the appropriate `ConfigureFolders` resource which is accessed via the `Configure...` button in the `Folders to Sort` section.).
